index.bs

```diff
@@ -10151,14 +10151,14 @@ used to configure various channel configurations.
 			set the channel count of the <var>node</var> output to
 			1 and return.
 
-			NOTE: For this case, the output chanel count will
-			change to <a>computedNumberOfChannels</a> dynamically
-			based on the input at runtime.
-
-		1. Otherwise set the channel counts of the outputs of the
+		1. Otherwise set the channel count of each output of the
 			<var>node</var> to 1 and return.
 </div>
 
+NOTE: For the case of the step 4.1 above, the output chanel count will
```

Comment:
This doesn't really help because this is actually normative.  The node MUST do this.  If you can't figure out a bikeshed way to make a note that isn't non-normative, just leave out "NOTE:" (and put it back where it was).

Reply:
No. I think this is informative note.

This algorithm describes what should happen when we validate the option. What happens at runtime is out of scope. That's why I used the "note" notation.

This has been a big problem in the spec. When a certain behavior is out of the algorithm scope, it should not be a step in the algorithm. Or make it clear that it is out of its scope.

Also having a note within an algorithm is not an illegal thing to do:
https://html.spec.whatwg.org/multipage/workers.html#worker-processing-model

If this were up to me, I would just put the note back into where it was.

Reply:
My point is that the way the algorithm is currently phrased the count is 1 and nothing else.  The note is that it can be changed from 1.  And if you leave the note out I take it to mean exactly one.  With the comment, then I see that it's not fixed to one.

But if you rephrase to say "initial count" is one, then you can leave the note in (or just remove it) because then I don't infer it to mean a fixed value of 1.

Reply:
That's a good idea. I'll use "initial count" and keep the note as is.

Reply:
Sounds good to me, fwiw.  I assume there is an explicit algorithm step somewhere else that changes the count to be not-1, right?

Reply:
Yes. The algorithm `computedNumberOfChannels` and mixing rules are rather well-defined.

Reply:
Sorry for not being clearer in my original comment!